scripts/create_native_image_test_env.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+
+# This script creates a submodule project with all the necessary repositories (forked) and changes needed to test
+# GraalVM updates with handwritten libraries (including, pubsub, bigquery, bigtable and spanner-jdbc).
+# It serves two main purposes; first, it allows for the creation of a testing environment that can be shared with
+# teammates and second, given that the submodule leverages creation of test branches in the (forked) repositories,
+# any extra configurations necessary for making the library compatible with GraalVM can easily be sent for review as a PR
+# from the branch itself.
+# At a high-level, the script will do the following:
+# - Modifies graal-sdk version in gapic-generator-java/gax
+# - Modifies the gapic-generator-bom version in java-shared-dependencies.
+# - Updates the version of native-maven-plugin in java-shared-config.
+# - Updates the java-shared-config and java-shared-dependencies versions in the handwritten libraries listed.
+# - Adds gapic-generator-java, java-shared-config and handwritten libraries listed previously
+#   with the associated changes, to the submodule project.
+set -eo pipefail
+
+function modify_shared_config() {
+  xmllint --shell pom.xml <<EOF
+  setns x=http://maven.apache.org/POM/4.0.0
+  cd .//x:artifactId[text()="google-cloud-shared-config"]
+  cd ../x:version
+  set ${SHARED_CONFIG_VERSION}
+  save pom.xml
+EOF
+}
+
+function modify_shared_dependencies() {
+  xmllint --shell pom.xml <<EOF
+  setns x=http://maven.apache.org/POM/4.0.0
+  cd .//x:artifactId[text()="google-cloud-shared-dependencies"]
+  cd ../x:version
+  set ${SHARED_DEPS_VERSION}
+  save pom.xml
+EOF
+}
+
+if [ -z "$GRAALVM_VERSION" ]; then
+  echo "Please specify the GRAALVM_VERSION."
+  exit 1
+fi
+
+if [ -z "$NATIVE_MAVEN_PLUGIN" ]; then
+  echo "Please specify the NATIVE_MAVEN_PLUGIN version you want to update to."
+  exit 1
+fi
+
+if [ -z "$ORGANIZATION" ]; then
+  echo "Please specify the ORGANIZATION where your updates need to be stored."
+  exit 1
+fi
+
+GRAALVM_BRANCH="${GRAALVM_VERSION}_update"
+
+## Round 1: Add gapic-generator-java and update graal-sdk version in GAX.
```

Comment:
It seems this script assumes this to run in a newly created repository with "git init"?

If so, can you add if statement to confirm the current working directory has ".git" directory?

Add source code comment about the assumption.

Reply:
Yeah it assumes that a repo is created with either 'git clone <link to remote>' or `git init` and `git remote add <link to remote>`.

Done.